modelling/src/main/java/org/axonframework/modelling/entity/SimpleEntityModel.java

```diff
@@ -213,10 +282,29 @@ public Builder<E> commandHandler(@Nonnull QualifiedName qualifiedName,
             return this;
         }
 
+        @Nonnull
+        @Override
+        public EntityModelBuilder<E> creationalCommandHandler(@Nonnull QualifiedName qualifiedName,
+                                                              @Nonnull CommandHandler messageHandler) {
+            requireNonNull(qualifiedName, "The qualifiedName may not be null.");
+            requireNonNull(messageHandler, "The messageHandler may not be null.");
+            if (creationalCommandHandlers.containsKey(qualifiedName)) {
+                throw new IllegalArgumentException(
```

Comment:
I think this should also throw a `DuplicateCommandHandlerSubscriptionException`.